packages/node_modules/@webex/internal-plugin-conversation/src/share-activity.js

```diff
@@ -122,42 +130,14 @@ const ShareActivity = WebexPlugin.extend({
      */
     return SCR.create()
       .then((scr) => {
-        scr.loc = gif.giphyURL;
+        scr.loc = gif.url;
         gifToAdd.scr = scr;
 
-        return Promise.resolve(
-          processImage({
```

Comment:
shouldn't need to processImage since giphy api already gives back a thumbnail with width/height